### PR TITLE
TW-2114: Display phone number as a clickable in message bubble

### PR DESCRIPTION
**Pull request by nqhhdev**
## Ticket
- #2114 

## Need merge

- https://github.com/linagora/flutter_matrix_html/pull/19
- https://github.com/nqhhdev/linkfy_text/pull/1

**Attach screenshots or videos demonstrating the changes**
- Web:
- Android:
- IOS:

### CODE REVIEW

File: lib/utils/url_launcher.dart [hunk 1]
```diff
@@ -73,7 +73,7 @@ class UrlLauncher with GoToDraftChatMixin {
           return;
         }
       }
-      launchUrlString(url!);
+      launchUrlString('https://$url');
```
Comment (tddang-linagora) [inline lib/utils/url_launcher.dart:76]: If `url` already have a scheme, will this work? E.g. "tel:123456" or "mailto:abc@example.com"

Comment (nqhhdev) [reply lib/utils/url_launcher.dart:76]: Don't worry about this case. In this case I was verified `url` with scheme if it was valid to return `url` if not only display text. You can see `RegExp` to verify `url` in here https://github.com/nqhhdev/linkfy_text/blob/9f4b9debbe4da621897ea040f24e66da3c9ba980/lib/src/utils/contants.dart#L1718

File: lib/pages/chat/chat_context_menu_actions.dart [hunk 2]
```diff
@@ -8,7 +8,8 @@ enum ChatContextMenuActions {
   pinChat,
   forward,
   downloadFile,
-  jumpToMessage;
+  jumpToMessage,
+  copyNumber;
```
Comment (hoangdat) [inline lib/pages/chat/chat_context_menu_actions.dart:12]: Should create `PhoneNumberContextMenu`

File: lib/utils/url_launcher.dart (hunk 1 above)
Comment (hoangdat) [reply lib/utils/url_launcher.dart:76]: still not clear to me @nqhhdev

File: lib/presentation/mixins/linkify_mixin.dart [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+    );
+    final selectedActionIndex = await showTwakeContextMenu(
+      context: context,
+      offset: offset,
+      listActions: listActions,
+      onClose: _handleStateContextMenu,
+    );
+    if (selectedActionIndex != null && selectedActionIndex is int) {
+      _handleClickOnContextMenuItem(
+        action: phoneNumberContextMenuOnWeb[selectedActionIndex],
```
Comment (hoangdat) [inline lib/presentation/mixins/linkify_mixin.dart:60]: why not `PhoneNumberContextMenuActions.copy`?

**Review by hoangdat — APPROVED** [review]

**Pull request merged**